pkg/util/indexedlist/indexedlist.go

```diff
@@ -101,7 +101,7 @@ func (il *IndexedList[K, V]) appendElement(e *element[K, V]) {
 
 // prependElement prepends a given element to the start of the list if not already present (anywhere in the list).
 // ATTENTION: The list must be locked when calling this function!
-func (il *IndexedList[K, V]) prependElement(e *element[K, V]) {
+func (il *IndexedList[K, V]) prependElement(e *element[K, V]) { //nolint:unused
```

Comment:
Just wondering, why does lint complain here?

Reply:
`prependElement` is not used anywhere and is not exported, that's why it complains.
I actually adapted this data structure from the "bucket" implementation in old ISS, where prepending was necessary. But since it was already implemented, it cost me nothing to keep it here too, and it makes the data structure more complete.